### PR TITLE
Add travis-ci build and badge.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
@etorreborre copied the version from saws, not sure if the scala version is ok?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: scala
+scala:
+   - 2.11.2
+jdk:
+  - oraclejdk7
+env:
+ - NO_FORK=true
+
+script: sbt -scala-version $TRAVIS_SCALA_VERSION 'test-only -- xonly exclude aws'
```
Comment (charleso) [inline .travis.yml]: No point excluding `aws` tags here I'm assuming - there won't be any in this project?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: scala
+scala:
+   - 2.11.2
+jdk:
+  - oraclejdk7
+env:
+ - NO_FORK=true
```
Comment (charleso) [inline .travis.yml]: This won't be used either - it's not defined in `build.scala` (see Ivory)

**Comment by etorreborre** [comment]
@tmcgilchrist we can use the latest Scala version which is `2.11.7`.

File: .travis.yml (hunk 1 above)
Comment (etorreborre) [inline .travis.yml]: Yes, no need to exclude `aws` here.

**Pull request merged**